docs/src/main/sphinx/language/reserved.rst

```diff
@@ -29,6 +29,7 @@ Keyword                        SQL:2016      SQL-92
 ``CURRENT_TIMESTAMP``          reserved      reserved
 ``CURRENT_USER``               reserved
 ``DEALLOCATE``                 reserved      reserved
+``DEFAULT``                    reserved      reserved
```

Comment:
`DEFAULT` isn't a reserved keyword in Trino at this time. That is why `docs-checks` job is failing.

Reply:
Maybe I'm misunderstanding, I thought that `DEFAULT` being an accepted keyword in, at least, the `... SET PROPERTIES foo = DEFAULT` context would mean that it's reserved?

Reply:
We generally try not to make keywords reserved unless there's a potential ambiguity. I don't think that's the case with `DEFAULT`. Also, making it reserved would likely cause backwards-compatibility issues.